Tests/Command/DropDatabaseDoctrineTest.php

```diff
@@ -3,14 +3,19 @@
 namespace Doctrine\Bundle\DoctrineBundle\Tests\Command;
 
 use Doctrine\Bundle\DoctrineBundle\Command\DropDatabaseDoctrineCommand;
+use Doctrine\DBAL\DBALException;
+use Generator;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Console\Application;
 use Symfony\Component\Console\Tester\CommandTester;
 
 class DropDatabaseDoctrineTest extends TestCase
 {
-    public function testExecute() : void
+    /**
+     * @dataProvider provideForceOption
+     */
+    public function testExecute(array $options) : void
```

Comment:
I've never tried this myself, but I think using variadics here could help simplify the data providers